bugbug/tools/code_review.py

```diff
@@ -314,57 +316,69 @@ def get_all_inline_comments(
         for revision in tqdm(phabricator.get_revisions(), total=revision_count):
             diff_comments: dict[int, list[InlineComment]] = defaultdict(list)
 
-            for transaction in revision["transactions"]:
-                if transaction["type"] != "inline":
-                    continue
-
-                # Ignore replies
-                if transaction["fields"]["replyToCommentPHID"] is not None:
-                    continue
-
-                # Ignore bot comments
-                if transaction["authorPHID"] == "PHID-USER-cje4weq32o3xyuegalpj":
-                    continue
-
-                if len(transaction["comments"]) != 1:
-                    # Follow up: https://github.com/mozilla/bugbug/issues/4218
-                    logger.warning(
-                        "Unexpected number of comments in transaction %s",
-                        transaction["id"],
+            try:
```

Comment:
Why are you wrapping everything in as a try-except block? This is very dangerous.

Reply:
I originally had this as a temporary solution when retrieving all inline comments, as I encountered the following error and it would halt the entire script:
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/bmah/Documents/GITHUB/bugbug/scripts/inline_comments_data_collection.py", line 153, in <module>
    download_inline_comments()
  File "/Users/bmah/Documents/GITHUB/bugbug/scripts/inline_comments_data_collection.py", line 59, in download_inline_comments
    for patch_id, comments in review_data.get_all_inline_comments(lambda c: True):
  File "/Users/bmah/Documents/GITHUB/bugbug/bugbug/tools/code_review.py", line 338, in get_all_inline_comments
    transaction_comment = transaction["comments"][0]
```
    
However, I've removed it in b9803344f026f9040f1ac3fb1e4ba48f5394004d as I have all the inline comments stored locally on my machine.

Reply:
I've added a `continue` for transactions with no comments here 063c4750d682f09a0cf9fba905e6ceef81a6e719. This should fix the issue!

Reply:
Removed in b68a1b33ce01854a6615c13bd321360b07adcc6b, as it has been resolved in #4266.